src/clr-angular/utils/drag-and-drop/interfaces/drag-event.ts

```diff
@@ -3,15 +3,18 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-
 export enum ClrDragEventType {
     DRAG_START,
     DRAG_MOVE,
     DRAG_END,
     DROP
 }
 
-export interface ClrDragEvent {
+export interface ClrDragEvent<T> {
     // TODO: this interface will be expanded and customized as the implementation progresses.
     type: ClrDragEventType;
+    draggableElement: Node;
+    dragPosition: {pageX: number, pageY: number};
```

Comment:
Should `DragPosition` be a separate type or interface? Feels like it will be reused in DnD a lot, no?